src/app/DTT/check_box/multi-check-box.js

```diff
@@ -1,8 +1,8 @@
 import 'app/DTT/check_box/styles.scss'
-import { Checkbox as CCheckbox, Stack } from '@chakra-ui/react'
+import { Checkbox as CCheckbox, HStack, Stack, Text, useTheme } from '@chakra-ui/react'
```

Comment:
Name it ChakraCheckbox rather than CCheckbox.

Reply:
Done, also renamed CText to ChakraText